sql/connect/server/src/main/scala/org/apache/spark/sql/connect/ml/MLUtils.scala

```diff
@@ -697,7 +703,12 @@ private[ml] object MLUtils {
       args: Array[Object],
       parameterTypes: Array[Class[_]]): Object = {
     validate(obj, methodName)
-    invokeMethod(obj, methodName, args, parameterTypes)
+    try {
+      invokeMethod(obj, methodName, args, parameterTypes)
+    } catch {
+      case e: InvocationTargetException =>
+        throw e.getCause
```

Comment:
the [InvocationTargetException](https://docs.oracle.com/javase/8/docs/api/java/lang/reflect/InvocationTargetException.html) docs say

> getCause()
> Returns the cause of this exception (the thrown target exception, which may be null).

Reply:
we should check null value